Sources/App/Github/GithubCommandRouter.swift

```diff
@@ -0,0 +1,42 @@
+import Vapor
+
+// Basic router implementation
+public class GithubCommandRouter {
```

Comment:
I'd name this `GithubClient`, `GithubHandler` or something of the likes

Reply:
I'm not a huge fan of either of those. It's not really a `GithubClient`. I'd use that name for something like what `GithubService` is. `GithubHandler` is a little vague. How about `GithubCommandHandler` or `GithubCommentHandler`?